Library/Homebrew/keg.rb

```diff
@@ -253,8 +253,13 @@ def python_pth_files_installed?
     Dir["#{path}/lib/python2.7/site-packages/*.pth"].any?
   end
 
+  def app_list(args = {})
+    app_prefix = (args[:prefer_opt] && optlinked?) ? opt_record : path
```

Comment:
Good question. I guess I wanted to stay as faithful as possible to the original implementation. But in reality `Keg#app_installed?` does not care which paths it is looking at. It makes sense to get rid of the argument and `args[:prefer_opt] &&` entirely.

Any opinion on `Dir[]` vs. `Pathname.glob()`? The latter seems to be more popular nowadays.